.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,5 +1,6 @@
 eclipse.preferences.version=1
```

Comment:
You shouldn't submit these files. Add them to the `.gitignore` so that the changes made to them won't be staged.